pkg/apis/tensorflow/v1alpha2/types.go

```diff
@@ -85,6 +95,20 @@ const (
 	CleanPodPolicyNone      CleanPodPolicy = "None"
 )
 
+// CleanPolicy defines the policy type of cleaning up tf-jobs.
+type CleanPolicy string
+
+const (
+	CleanPolicyUndefined   CleanPolicy = ""
+	CleanPolicyAllPods     CleanPolicy = "AllPods"
+	CleanPolicyRunningPods CleanPolicy = "RunningPods"
+	CleanPolicyJob         CleanPolicy = "Job"
+	CleanPolicyNone        CleanPolicy = "None"
+)
+
+// CleanDelay defines the delay of executing CleanPolicy.
+type CleanDelay string
```

Comment:
copy the comment from this [link](https://github.com/golang/go/blob/master/src/time/format.go#L1274) to describe the format of the string would be useful